app/controllers/sales_controller.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+class SalesController < ApplicationController # :nodoc:
+  include SimpleCrudController
+
+  private
+
+  def render_show(existing_sale)
```

Comment:
Is the method in the `SimpleCrudController` also called `existing_resource` or something... I'd just stick with sale here if I were you. :smile: